src/plugins/abrt-action-analyze-core.in

```diff
@@ -114,14 +97,6 @@ def extract_info_from_core(coredump_name):
     log1("Found %i libs" % len(libraries))
     return build_ids
 
-def build_ids_to_path(build_ids):
```

Comment:
Oh, seems that this comes from bc470449aebc700ec118ea4de84581d1b3afc3bc. Never used. Tsk tsk.